libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/PushNotificationManager.swift

```diff
@@ -235,11 +245,11 @@ extension PushNotificationManager {
                                                notificationId: String,
                                                actionIdentifier: String
     ) async throws -> ActionResultRepresentation {
-        guard client.apiVersion.compare("v61.0").rawValue >= 0 else {
-            throw PushNotificationManagerError.notificationActionInvocationFailed("API Version must be at least v61.0")
+        guard client.apiVersion.compare("v64.0").rawValue >= 0 else {
+            throw PushNotificationManagerError.notificationActionInvocationFailed("API Version must be at least v64.0")
         }
         
-        let path = "/connect/notifications/\(notificationId)/actions/\(actionIdentifier)"
+        let path = "\(client.apiVersion)/connect/notifications/\(notificationId)/actions/\(actionIdentifier)"
```

Comment:
Same case with API Version here.  Wasn't seeing it added on request when it was sent out, so manually adding it here.